elements/rh-band/demo/index.html

```diff
@@ -0,0 +1,152 @@
+      <rh-cta priority="primary" slot="footer">
+        <a href="#">Learn more</a>
+      </rh-cta>
+      <div slot="aside">
+        <rh-card color="complement">
+          <h3 slot="header">Lorem ipsum</h3>
+          <p>Ut wisi enim ad minim veniam.</p>
+          <rh-cta slot="footer" priority="tertiary" on="dark">
+            <a href="#">Learn more</a>
+          </rh-cta>
+        </rh-card>
+      </div>
+    </rh-band>
+
+    <rh-band aside="right full bottom">
+      <h2 slot="header">Default band</h2>
+      <p slot="header">Consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua.</p>
+      <p>Consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At vero eos et accusam et justo duo dolores et ea rebum. Stet clita kasd gubergren, no sea takimata sanctus est Lorem ipsum dolor sit amet. Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At vero eos et accusam et justo duo dolores et ea rebum. Stet clita kasd gubergren, no sea takimata sanctus est Lorem ipsum dolor sit amet. Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At vero eos et accusam et justo duo dolores et ea rebum. Stet clita kasd gubergren, no sea takimata sanctus est Lorem ipsum dolor sit amet.</p>
+      <rh-cta priority="primary" slot="footer">
+        <a href="#">Learn more</a>
+      </rh-cta>
+      <div slot="aside">
+        <rh-card color="dark">
+          <h3 slot="header">Lorem ipsum</h3>
+          <p>Ut wisi enim ad minim veniam. Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At vero eos et accusam et justo duo dolores et ea rebum.</p>
+          <rh-cta slot="footer" priority="tertiary" on="dark">
+            <a href="#">Learn more</a>
+          </rh-cta>
+        </rh-card>
+      </div>
+    </rh-band>
+
+    <rh-band color="dark" aside="left full bottom" img-src="https://www.redhat.com/profiles/rh/themes/redhatdotcom/img/header/header-hub-svsp-2000x1562.jpg">
+      <h2 slot="header">Dark band with imaged background</h2>
```

Comment:
On all these demos, can we add the value of the aside attribute to the header text in the band, to ensure the visuals match up with the expected layout?